### DISCUSSION TITLE
Hit localhost from a batect-spun container? :)

### DISCUSSION THREAD

**Discussion by adameichelkraut** [batect/batect#1461] [Q&A]
TLDR; I have a separate web server that might be running in localhost that I wish to hit with my batect-spun-up container! Anyone have any snippets for how to do this properly?

Tried sifting lightly through the discussions -- wondered if this was already answered. I also did some light googling on the topic at large within the Docker-verse and it seems folks have found luck by either setting the network to `host` and fiddling with setting some `additional_hosts` on the container to point the container to the host machine's network.

I've tried using `--use-network=host` with batect, however I get:
```
Error response from daemon: network-scoped alias is supported only for containers in user defined networks
```

I've also tried adding something like this to the container's config:
```
    ...
    additional_hosts:
      host.docker.internal: host-gateway
```
However, I'm thinking that this above needs to be paired with the use network host.

Maybe I'm missing something in the docs? Look forward to chatting! Thanks!

**Comment by charleskorn** [comment]
You should be able to use `host.docker.internal` on Mac and Windows hosts (and possibly Linux hosts too) without any configuration or switching to host networking. What happens when you try to use this hostname?

One thing to check: the target web server on your machine may need to be listening on all local addresses, not just the loopback address. (ie. check if it is listening on `0.0.0.0`, not `127.0.0.1`). 

Similarly, I'm not sure if `host.docker.internal` supports IPv6, so check if your web server is listening on IPv6 or IPv4.
